MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -119,19 +120,21 @@ CompletionContext CreateCompletionContext (List<CompletionItem> items)
 
 			foreach (var att in rr.GetAttributeCompletions (doc, doc.ToolsVersion)) {
 				if (!existingAtts.ContainsKey (att.Name)) {
-					items.Add (CreateCompletionItem (att));
+					items.Add (CreateCompletionItem (att, MSBuildSpecialCommitKind.Attribute, XmlCompletionItemKind.Attribute));
 				}
 			}
 
 			return CreateCompletionContext (items);
 		}
 
-		CompletionItem CreateCompletionItem (BaseInfo info, string prefix = null)
+		CompletionItem CreateCompletionItem (BaseInfo info, MSBuildSpecialCommitKind mSBuildSpecialCommitKind, XmlCompletionItemKind xmlCompletionItemKind, string prefix = null)
```

Comment:
I don't think any of the MSBuildSpecialCommitKind values assigned via this method are actually used so the parameter can be removed

Reply:
removed